Automattic/wp-calypso
# Set dismissible cards on domain-only upsell carousel

## Changes proposed in this Pull Request

This PR add the option to dismiss domain only upsell carousel's cards for 1 month or 1 week.

It's part of the new domain-only flow (pcYYhz-ye-p2)

![carousel-close](https://user-images.githubusercontent.com/2797601/154066380-dd9ef4f3-0b64-4d06-b0b2-76a857fa860d.png)

![carousel-open](https://user-images.githubusercontent.com/2797601/154066375-a359fb14-2d68-459c-ad4e-be2df19f9997.png)

## Testing instructions

_Prerequisites: you need a domain-only subscriptions to test this PR_

- Build this branch locally or open the live Calypso link
- Navigate to to `/manage/domains`
- Click on the "Hide this" button on one of the carda and verify that it's removed.
- You can reactivate the card through the Developer console, using this endpoint `/me/domains/[:domainName]/notices/[:noticeType]/message`, where `noticeType` is `upsellCarouselHideCreateSiteCardUntil` or `upsellCarouselHideAddEmailCardUntil`
